src/app/commands/cmd_change_pixel_format.cpp

```diff
@@ -673,32 +595,32 @@ void ChangePixelFormatCommand::onExecute(Context* context)
       tx(new cmd::FlattenLayers(sprite, selLayers, options));
     }
 
-    job.startJobWithCallback([this, &job, sprite](Tx& tx) {
+    job.startJobWithCallback([this, &job, sprite, &matrix](Tx& tx) {
       tx(new cmd::SetPixelFormat(sprite,
-                                 m_format,
-                                 m_dithering,
-                                 m_rgbmap,
-                                 get_gray_func(m_toGray),
+                                 params().format(),
+                                 render::Dithering(params().dithering(), matrix, params().factor()),
+                                 params().rgbmap(),
+                                 get_gray_func(params().toGray()),
                                  &job,
-                                 m_fitCriteria)); // SpriteJob is a render::TaskDelegate
+                                 params().fitCriteria())); // SpriteJob is a render::TaskDelegate
```

Comment:
Not introduced in this PR, but introduced here https://github.com/aseprite/aseprite/commit/3cc1c63274dac90360b8194f66101f86f1a670b6#diff-6a0da30ae1d087883f14800a3979cf9ffbab2e035cc9cc23b1fede5d83278046L682, the `SpriteJob` comment is for the `&job` line. We can fix this in this commit.